quipucords/scanner/network/connect.py

```diff
@@ -162,11 +162,12 @@ def run_with_result_store(self, result_store):
         serializer = SourceSerializer(self.scan_task.source)
         source = serializer.data
 
-        if self.scan_job.options is None:
-            forks = ScanOptions.get_default_forks()
-        else:
+        if self.scan_job.options is not None:
```

Comment:
Out of curiosity, is there a reason we switched this if..else?

Reply:
It was done that way in another file so I switched it for consistency. Doesn't really matter.